augur/filter.py

```diff
@@ -82,6 +82,8 @@ def run(args):
 
     if args.cat and args.viruses_per_cat:
         vpc = args.viruses_per_cat
+        if vpc == 0:
```

Comment:
This doesn't seem necessary. The default value for `viruses_per_cat` is `None`, so the user can skip subsampling by omitting the `--viruses_per_cat` flag in the Snakefile. If they specify a value of 0, the preceding if statement should evaluate to false and use the complete data set.